dts/arm/st/stm32f7.dtsi

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright (c) 2018 Yurii Hamann
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv7-m.dtsi>
+#include <st/stm32f7-pinctrl.dtsi>
+#include <st/mem.h>
+#include <dt-bindings/clock/stm32_clock.h>
+#include <dt-bindings/gpio/gpio.h>
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m7";
+			reg = <0>;
+		};
+	};
+
+	sram0: memory@20000000 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+		reg = <0x20000000 DT_SRAM_SIZE>;
+	};
+
+	flash0: flash@8000000 {
+		compatible = "soc-nv-flash";
+		label = "FLASH_STM32";
+		reg = <0x08000000 DT_FLASH_SIZE>;
+		write-block-size = <1>;
+	};
+
+	soc {
+		rcc: rcc@40023800 {
+			compatible = "st,stm32-rcc";
+			clocks-controller;
+			#clock-cells = <2>;
+			reg = <0x40023800 0x400>;
+			label = "STM32_CLK_RCC";
+		};
+
+		pinctrl: pin-controller@40020000 {
+			compatible = "st,stm32-pinmux";
+			#address-cells = <1>;
+			#size-cells = <1>;
+			reg = <0x48000000 0x2400>;
```

Comment:
reg = <0x40020000 0x8000>;

Reply:
Why size is 0x8000 for pin-controller? As I understand it's size of all GPIO sections. 8 GPIO sections has size 0x2400.

Reply:
My bad, I mixed pinmux and uart address.
2400 is fine. Removing my comment. Thanks